doc/whats_new/upcoming_changes/array-api/30819.feature.rst

```diff
@@ -0,0 +1,2 @@
+- :func:`sklearn.utils.extmath.randomized_svd` now support Array API compatible inputs.
+  By :user:`Connor Lane <clane9>`.
```

Comment:
Generous attribution to user @clane9 😂

Reply:
Maybe @jeremiedbb you could add yourself as a co-author of this contribution ;)